src/api/emit.js

```diff
@@ -10,11 +10,16 @@ const CustomEvent = ((Event) => {
 })(window.CustomEvent);
 
 function createCustomEvent(name, opts = {}) {
-  if (CustomEvent) {
-    return new CustomEvent(name, opts);
+  let e;
+  if (Event) {
+    e = new Event(name, opts);
```

Comment:
I think not having to maintain a whitelist is reasonable here. If anything, we should see a red build if something isn't working right.